### PR TITLE
[JFG-895]-Jagger uses one-by-one load balancer instead of round robin.

### PR DESCRIPTION
**Pull request by SokolAndrey**

### CODE REVIEW

File: chassis/spring.schema/src/main/java/com/griddynamics/jagger/xml/beanParsers/workload/balancer/RoundRobinBalancerDefinitionParser.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package com.griddynamics.jagger.xml.beanParsers.workload.balancer;
 
 import com.griddynamics.jagger.invoker.OneByOnePairSupplierFactory;
```
Comment (dlatnikov) [inline chassis/spring.schema/src/main/java/com/griddynamics/jagger/xml/beanParsers/workload/balancer/RoundRobinBalancerDefinitionParser.java]: Please optimize import

**Comment by anton-antonenko** [comment]
:+1:

**Comment by dlatnikov** [comment]
👍

**Comment by a-badaev** [comment]
+1

**Pull request merged**